### ISSUE TITLE
[BUG] Archive `ide-typescript`

### ISSUE THREAD

**Issue by AlexWayfer** [pulsar-edit/package-backend#258]
**What is the Bug**

It seems working. Kind of, for example it marks my JavaScript (not TypeScript) code wrong:

![image](https://github.com/pulsar-edit/package-backend/assets/6510020/102b236e-aea8-4ea1-8076-24c6f00637e8)

But it's still the archived repository: https://github.com/atom/ide-typescript

![image](https://github.com/pulsar-edit/package-backend/assets/6510020/0a5388db-e508-402e-9d54-7de69983940a)

[You've wrote](https://github.com/pulsar-edit/package-backend/issues/248#issuecomment-1996747391) that `pulsar-ide-typescript-alpha` can replace it.

**Comment by savetheclocktower** [comment]
Yeah, I figure this meets the definition of “Archived” by our standards — not broken, but clearly orphaned.

I think it's probably about time to think about renaming `pulsar-ide-typescript-alpha` to `pulsar-ide-typescript` — at the time I didn't want it to seem like I was squatting on the name. Either way, I'm fine with the admin action message mentioning it as a possible replacement.

**Comment by confused-Techie** [comment]
I wanted to go ahead and be clear and in no uncertain terms. @savetheclocktower Would you like me to point to `pulsar-ide-typescript-alpha` as the alternative to `ide-typescript` once marked as archived? Would you like me to wait to do so until you've felt you've ironed out the edges and renamed it? Or should I mark as archived here, and pointing to yours instead can come at a later date & time.
